airflow-core/src/airflow/cli/cli_config.py

```diff
@@ -812,6 +812,18 @@ def string_lower_type(val):
     help="The option name(s) to ignore to update in the airflow config.",
     type=string_list_type,
 )
+ARG_UPDATE_CONFIG_FIX = Arg(
+    ("--fix",),
+    help="Automatically apply the configuration changes instead of performing a dry run. (Default: dry-run mode)",
+    action="store_true",
+)
+
+ARG_UPDATE_ALL_RECOMMENDATIONS = Arg(
+    ("--all-recommendations",),
+    help="Include non-breaking (recommended) changes along with breaking ones. (Use with --fix or --dry-run)",
```

Comment:
But the `--dry-run` flag has been removed below?

Reply:
Ohh I missed this. dry run is there by default